Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Domain/Model/LocalStorageContext.cs

```diff
@@ -2,23 +2,22 @@
 
 namespace MvvmSeed.Domain.Model
 {
-    public class LocalStorageContext : DbContext
+    public class LocalStorageContext : DbContextBase
     {
-        private readonly string _databasePath;
+        private const string DatabaseName = "LocalStorage.db";
 
+        /// <summary>
+        /// Initializes a InMemory database context to be used on UnitTests
+        /// </summary>
+        /// <param name="optionsBuilder"><see cref="DbContextOptions"/> specifying 'UseInMemoryDatabase' option</param>
         public LocalStorageContext(DbContextOptions optionsBuilder) : base(optionsBuilder) { }
 
-        public LocalStorageContext(string databasePath)
-        {
-            _databasePath = databasePath;
-        }
+        /// <summary>
+        /// Initializes a Sqlite database context
+        /// </summary>
+        /// <param name="databaseFolder">Full path where the database will be created/open </param>
+        public LocalStorageContext(string databaseFolder) : base(databaseFolder, DatabaseName) { }
```

Comment:
Yep, I prefer to hide that from consumers.. could be worth it on some scenarios, but could be error-prone too. So, again, I'll let it evolve naturally, if the need arises, we'll add the parameter for db name.